vault/identity_store_oidc_test.go

```diff
@@ -937,12 +937,32 @@ func TestOIDC_PeriodicFunc(t *testing.T) {
 				numPublicKeys int
 			}{
```

Comment:
Nit: I noticed that cycle is pretty much the `index+1` for this slice. Might be a good opportunity to clean this up a bit and turn the struct into an internal type, something like `[]cycle{...}`.